doc/en/fixture.rst

```diff
@@ -804,48 +804,61 @@ without declaring a function argument explicitly or a `usefixtures`_ decorator.
 As a practical example, suppose we have a database fixture which has a
 begin/rollback/commit architecture and we want to automatically surround
 each test method by a transaction and a rollback.  Here is a dummy
-self-contained implementation of this idea::
+self-contained implementation of this idea in a style resembling xUnit setup
+and tearDown methods::
 
-    # content of test_db_transact.py
+    # content of test_foo.py
 
     import pytest
+    import MyFoo
```

Comment:
Please note that this code will be run by `regendoc`, so my guess is that this will fail. Check by running `tox -e regen` in a clean checkout.